tools/image_processing/yolo/doclayoutyolo.xml

```diff
@@ -0,0 +1,120 @@
+<tool id="doclayout_yolo" name="DocLayout-YOLO: Enhancing Document Layout Analysis" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>through Diverse Synthetic Data and Global-to-Local Adaptive Perception</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.0.4.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@PROFILE@">23.2</token>
+    </macros>
+    <creator>
+        <organization name="European Galaxy Team" url="https://galaxyproject.org/eu/"/>
+        <person givenName="Anup" familyName="Kumar" email="kumara@informatik.uni-freiburg.de"/>
+    </creator>
```

Comment:
Can you please add main.py (maybe rename it to something more specific) as https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files

Reply:
Done!